docs/framework/process_entrypoint.md

```diff
@@ -15,14 +25,5 @@ This concept allows to shutdown gracefully in any case of error or termination r
 
 On NAO and in Webots the robotic control software needs access to the hardware or simulator interface.
 The hardware interface provides an abstract way to interact with the underlying backend.
-The `main()` function first initializes the hardware interface and then constructs the runtime with it.
+The `main()` function first initializes the hardware interface and then starts the runtime (`run()`) with it.
 See [Hardware Interface](./hardware_interface.md) for more information about what the hardware interface initializes.
-At the end, the runtime is started.
-The `main()` function then waits for termination of the runtime which then concludes the process execution.
-
-## Behavior Simulator
-
-The behavior simulator is a special build target which only initializes and starts a subset of the robotic control software.
-It is intended to be executed on the development machine.
-Cancellation and hardware interfaces are not needed and are therefore omitted from initialization in `main()`.
-Instead, the behavior simulator parses command line arguments and dispatches the behavior simulation.
```

Comment:
Why did you remove this?

Reply:
Iirc the behavior simulator is not a real HULK binary anymore. It works very differently currently. Therefore it would look weird if it would be included here.